cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocProvisioner.java

```diff
@@ -127,14 +128,16 @@ public Cluster createCluster(ProvisionerContext context) throws Exception {
       }
 
       // Reload system context properties and get system labels
-      Map<String, String> systemLabels = getSystemLabels();
-      LOG.info("Creating Dataproc cluster {} in project {}, in region {}, with image {}, with system labels {}",
-               clusterName, conf.getProjectId(),
-               conf.getRegion(), imageDescription, systemLabels);
+      Map<String, String> labels = ImmutableMap.<String, String>builder()
+        .putAll(getSystemLabels())
+        .putAll(conf.getClusterLabels())
```

Comment:
if there is a key in both the system and cluster labels, does this throw an exception? I seem to remember ImmutableMap disallowing it, but not sure.

Reply:
good catch, thanks!